app/src/main/java/com/kickstarter/services/transformers/GraphQLTransformers.kt

```diff
@@ -874,7 +874,6 @@ fun getCreateAttributionEventMutation(eventInput: CreateAttributionEventData, gs
             .eventName(eventInput.eventName)
             .eventProperties(eventPropertiesJson)
             .projectId(eventInput.projectId)
-            .clientMutationId(eventInput.clientMutationId)
```

Comment:
Pretty sure the graphql clients generate these on request, rather than needing to be specified. Regardless, the mutation works fine without me specifying it.

Reply:
yeah, we never really use it so should be fine to remove